pkg/rpc/context_test.go

```diff
@@ -215,7 +248,10 @@ func TestHeartbeatHealth(t *testing.T) {
 
 	clientCtx.SetLocalInternalServer(&internalServer{})
 
-	if err := clientCtx.ConnHealth(clientCtx.Addr); err != nil {
+	if err := clientCtx.ConnHealth(clientCtx.Addr); err != ErrNotConnected {
+		t.Error(err)
```

Comment:
Switch this to a `t.Errorf` and explain that the error should have been `ErrNotConnected`, otherwise, if it returns no error, the test failure will be very unclear.